packages/sample-app/sample_app/methods_decorated_app.py

```diff
@@ -10,65 +10,32 @@
 Traceloop.init(app_name="joke_generation_service")
 
 
-@task(name="joke_creation", version=1)
```

Comment:
Why did you delete this?

Reply:
Actually I copied this file from Gal branch, I just saw he excluded it from his merged PR, so I'll remove this change as well 🙈